project-rome-docs/objectivec-api/remotesystems/index.md

```diff
@@ -1,25 +1,31 @@
-# Discovery namespace
+# RemoteSystems namespace
 ```
-discovery.h
+ConnectedDevicesRemoteSystems.h
 ```
 
-The Discovery namespace contains APIs that allow a client app to discover information about remote devices and applications.
+The RemoteSystems namespace contains APIs that enable a client app to send commands to remote devices and applications.
```

Comment:
The previous text was a more accurate description of what's in the `RemoteSystems` namespace.